op-supervisor/supervisor/backend/cross/hazard_set.go

```diff
@@ -97,7 +101,7 @@ func (h *HazardSet) checkMessageWithCurrentTimestamp(msg *types.ExecutingMessage
 	existing, ok := h.entries[msg.Chain]
 	if ok {
 		if existing.ID() != includedIn.ID() {
-			return true, fmt.Errorf("found dependency on %s (chain %d), but already depend on %s", includedIn, initChainID, existing)
+			return true, fmt.Errorf("found dependency on %s (chain %d), but already depend on %s: %w", includedIn, initChainID, existing, types.ErrConflict)
```

Comment:
this is the only change that possibly alters the behavior of the supervisor. There aren't any tests asserting this case and I'm not sure whether this is truly a conflict.

Reply:
Thanks for confirming. I've updated the tests to check for ErrConflicts in this case.